usage.ipfs.js

```diff
@@ -0,0 +1,76 @@
+/*!
+ * Copyright(c) ConsenSys Software Inc.
+ * Copyright(c) https://consensys.net/
+ * MIT Licensed
+ */
+
+/* eslint-disable */
+
+import { config as loadEnv } from 'dotenv';
+import path from 'path';
+import { SDK, Auth, TEMPLATES, Metadata } from './index.js';
+loadEnv();
+
+const projectId = process.env.INFURA_IPFS_PROJECT_ID;
+const projectSecret = process.env.INFURA_IPFS_PROJECT_SECRET;
+const ipfsUrl = process.env.INFURA_IPFS_ENDPOINT;
+
+const myMetadataStorage = new Metadata({ projectId, projectSecret, ipfsUrl });
+
+/**
+ * METADATA
+ */
+// CREATE CONTRACT URI
+// upload collection metadata from JSON object
+const collectionMetadata = {
+  name: 'My awesome collection',
+  description: "A long description explaining why it's awesome",
+  image: path.join(__dirname, 'test', 'ipfs-test/consensys.png'), // accepts absolute path and URL
+  external_link: 'https://myawesomewebsite.net',
+};
+const { cid } = await myMetadataStorage.createContractURI(collectionMetadata);
+
+// upload collection metadata from JSON file
+const myjson = path.join(__dirname, 'test', 'ipfs-test/collectionMetadata.json');
+const contractURI_file = await myMetadataStorage.createContractURI(myjson);
+
+// CREATE METADATA URI
+// upload token metadata from JSON object
+const tokenMetadata = {
+  description: 'Friendly OpenSea Creature that enjoys long swims in the ocean.',
+  external_url: 'https://openseacreatures.io/3',
+  image: 'https://storage.googleapis.com/opensea-prod.appspot.com/puffs/3.png',
+  animation_url: path.join(__dirname, 'test', 'ipfs-test/consensys.png'),
+  name: 'Dave Starbelly',
+  attributes: [],
+};
+const { cid: cidFile } = await myMetadataStorage.createTokenURI(tokenMetadata);
+
+// upload token metadata from JSON file
+const mynftjson = path.join(__dirname, 'test', 'ipfs-test/nftMetadata.json');
+const tokenURI_file = await myMetadataStorage.createTokenURI(mynftjson);
+
+/**
+ *
+ * SDK
+ *
+ */
+const acc = new Auth({
```

Comment:
what about Dayan's suggestion?
const authETH = new Auth({
  network: {
      projectId: process.env.INFURA_PROJECT_ID,
      secretId: process.env.INFURA_PROJECT_SECRET,
      privateKey: process.env.WALLET_PRIVATE_KEY,
      rpcUrl: process.env.EVM_RPC_URL,
      chainId: 5, // Goerli
    },
    ipfs: {
      projectId: process.env.IPFS_PROJECT_ID,
      secretId: process.env.IPFS_PROJECT_SECRET,
      ipfsUrl?: process.env.IPFS_GATEWAY, // default to ipfs.infura.io:5001
    }
  }